Storage and flow states with custom Jacobian + linear solve: the formulation to rule them all (#3150)
Deltares/Ribasim: docs/reference/usage.qmd

The solver section in the configuration file is entirely optional, since we aim to use defaults that will generally work well.
Common reasons to modify the solver settings are to adjust the calculation or result stepsizes: `dt`, and `saveat`.

> If your model does not converge, or your performance is lower than expected, it can help to adjust other solver settings as well. For example, setting `optimized_implicit_solve = false` will bypass much of the custom numerical code written for Ribasim, using a less optimized but potentially more robust algorithm.

Reviewer: I'd leave this out here, having it below is enough, and 'potentially more robust' is hard to grasp for users I think.

```suggestion
If your model does not converge, or your performance is lower than expected, it can help to adjust other solver settings as well.
```

The `link_id` value is the same as the `fid` written to the Link table, and can be used to directly look up the Link geometry.
Flows from the "from" to the "to" node have a positive sign, and if the flow is reversed it will be negative.
The `convergence` indicates how hard the flow over this link was to solve compared to the other flows, see [convergence output](#sec-convergence).

## Convergence output {#sec-convergence}

> Both the Basin and the flow results contain a `convergence` column, which indicates which parts of the model are hardest for the solver to solve.

Reviewer: ```suggestion
Both the Basin and the flow results contain a `convergence` column, which indicates which parts of the model are hardest on the solver.
```
